Tribes/src/app/mock-building.ts

```diff
@@ -0,0 +1,8 @@
+import { Building } from './building';
+
+export const BUILDINGS: Building[] = [
+    {id: 1, type: 'townhall', level: 1, hp: 4, startedAt: 123, finishedAt: 456, imageURL: 'assets/images/building.png'},
+    {id: 2, type: 'barrak', level: 1, hp: 4, startedAt: 123, finishedAt: 456, imageURL: 'assets/images/barrak.png'},
```

Comment:
`barracks` is the term, I think it doesn't have 'egyes szama''

Reply:
Corrected